src/create-dom-event.ts

```diff
@@ -9,11 +9,47 @@ interface TriggerOptions {
 
 interface EventParams {
   eventType: string
-  modifier: string
-  meta: any
+  modifiers: string[]
   options?: TriggerOptions
 }
 
+// modifiers to keep an eye on
+const ignorableKeyModifiers = ['stop', 'prevent', 'self', 'exact', 'exact']
```

Comment:
Where did this list come from? somewhere in Vue 3?

Reply:
Yeah, from the vOn transformer, they are used to wrap the callback